### PR TITLE
1.1 Simple functions

### PR DESCRIPTION
**Pull request by AlexandraPopovaz**

### CODE REVIEW

File: src/usualFunctions.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+};
+divide(10, 2);
+
+//3. multiplication
+const mul = function (f: number, p: number) {
+    return console.log(f * p);
+};
+mul(2, 2);
+
+export {divide, mul};
```
Comment (viraxslot) [inline src/usualFunctions.ts:19]: usually we export at the end of the file, easier to find

File: src/usualFunctions.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const minus = function (q: number, w: number) {
+    return console.log(q - w);
+};
+minus(10, 1);
+
+//5. One function with 3 values from previous functions
+const sum3 = (a:number, b:number) => a + b;
+const divide3 = (x: number, y: number) => x / y;
+
+let math = {
```
Comment (viraxslot) [inline src/usualFunctions.ts:31]: Nope. Common function -> in its body you should call "sum", "divide", "mul" -> later in the same body return an object (no console logs) -> in the main script call the function and save the returned value to some variable -> use console.log and print all values.

Usually, function doesn't log the data. It should be logged in the place of calling the function.

File: src/arrowFunctions.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import {sum} from './usualFunctions'
```
Comment (viraxslot) [inline src/arrowFunctions.ts:1]: The file should look like:
imports
commands
exports

Please don't mix them.

File: src/arrowFunctions.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+import {divide, mul} from './usualFunctions'
+divide(1, 2);
+mul(1, 2);
+
+//1-long version with 2 parameters
+const sum2 = (a: number, b: number) => {
+    return console.log(a + b);
+};
+sum2(1, 1);
```
Comment (viraxslot) [inline src/arrowFunctions.ts:12]: Let's make 1 file with all function descriptions and the second where you use them.

File: src/usualFunctions.ts (hunk 1 above)
Comment (AlexandraPopovaz) [reply src/usualFunctions.ts]: Thanks! Fixed

File: src/arrowFunctions.ts (hunk 3 above)
Comment (AlexandraPopovaz) [reply src/arrowFunctions.ts]: Thanks! Fixed files

File: src/arrowFunctions.ts (hunk 4 above)
Comment (AlexandraPopovaz) [reply src/arrowFunctions.ts]: Ok, fixed

File: src/setupF.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
+export const sum = function (a:number, b:number) {
+    return a + b;
+};
+
+export const divide = function (x: number, y: number) {
```
Comment (viraxslot) [inline src/setupF.ts:5]: I'd use a and b everywhere, but it's not necessary of course.

**Review by viraxslot — APPROVED** [review]

**Pull request merged**